gateways/authorizenet/authorizenet.go

```diff
@@ -121,6 +128,38 @@ func (client *AuthorizeNetClient) Refund(request *sleet.RefundRequest) (*sleet.R
 	}, nil
 }
 
+func shouldVoidUnsettledCapture(requestOptions map[string]interface{}) bool {
+	if requestOptions[shouldVoidUnsettledCaptureOption] != nil {
+		shouldVoidUnsettledCapture, ok := requestOptions[shouldVoidUnsettledCaptureOption].(bool)
+		if ok {
+			return shouldVoidUnsettledCapture
+		}
+	}
+	return false
+}
+
+func sendVoidFromRefundRequest(client *AuthorizeNetClient, request *sleet.RefundRequest) (*sleet.RefundResponse, error) {
+	voidRequest := sleet.VoidRequest{
+		TransactionReference:       request.TransactionReference,
+		ClientTransactionReference: request.ClientTransactionReference,
+		MerchantOrderReference:     request.MerchantOrderReference,
+	}
+
+	authorzeNetVoidResponse, err := client.Void(&voidRequest)
+	if err != nil {
+		return nil, err
+	}
+
+	if authorzeNetVoidResponse.ErrorCode != nil {
+		return &sleet.RefundResponse{ErrorCode: authorzeNetVoidResponse.ErrorCode}, nil
```

Comment:
Should we also pass Success: False?

Reply:
This will default to false and seems to be the standard way we structure this.